### PR TITLE
add Windows support

### PR DESCRIPTION
**Pull request by SaltyCharles**
Currently only pulling content from secedit.  Additional modules to
audit the full CIS stack coming soon.

### CODE REVIEW

File: hubblestack_nova/secedit.py [hunk 1]
```diff
@@ -0,0 +1,249 @@
+#secedit.py
+# -*- encoding: utf-8 -*-
+'''
+Loader and primary interface for nova modules
```
Comment (basepi) [inline hubblestack_nova/secedit.py:4]: copy pasta

File: hubblestack_nova/secedit.py [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+    import codecs
+    HAS_WINDOWS_MODULES = True
+except ImportError:
+    HAS_WINDOWS_MODULES = False
+
+log = logging.getLogger(__name__)
+__virtualname__ = 'secedit'
+
+def __virtual__():
+    if not salt.utils.is_windows():
```
Comment (basepi) [inline hubblestack_nova/secedit.py]: should this also check `HAS_WINDOWS_MODULES` ?

File: hubblestack_nova/secedit.py [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+    Merge two yaml dicts together at the secedit:blacklist and
+    secedit:whitelist level
+    '''
+    if 'secedit' not in ret:
+        ret['secedit'] = {}
+    for topkey in ('blacklist', 'whitelist'):
+        if topkey in data.get('secedit', {}):
+            if topkey not in ret['secedit']:
+                ret['secedit'][topkey] = {}
+            ret['secedit'][topkey].update(data['secedit'][topkey])
```
Comment (basepi) [inline hubblestack_nova/secedit.py]: I recommend changing this slightly. This is the general format that I used to use for the other modules, but the problem is if you have the same top-level key in two different yaml files, whichever one you parse last will overwrite the others. I switched to using a list like https://github.com/HubbleStack/Nova/blob/3de386e99734af9758a0870a0d9509c4e7df4b0e/hubblestack_nova/pkg.py#L215-L218, though that also requires some changes to _get_tags as well. You can see the way I changed all the modules in my pull request https://github.com/HubbleStack/Nova/pull/91/files

The changes look bigger than they are. In `_get_tags` you basically just need one more nested loop to iterate over the list of tag data. Keep me posted if you have questions.

**Comment by basepi** [comment]
Left a few comments in line, otherwise this looks awesome!

File: hubblestack_nova/secedit.py (hunk 2 above)
Comment (SaltyCharles) [inline hubblestack_nova/secedit.py]: can probably put the whole block under `__virtual__`

Comment (basepi) [inline hubblestack_nova/secedit.py]: Nah, the imports should stay above. I was thinking like this:

```
if not salt.utils.is_windows() or not HAS_WINDOWS_MODULES:
```

Comment (SaltyCharles) [inline hubblestack_nova/secedit.py]: that works better

**Comment by basepi** [comment]
Awesome, thanks!

**Pull request merged**